Makefile

```diff
@@ -112,6 +112,16 @@ lint-fix: ## Fix linting issues automagically
 #	@go run -modfile tools/go.mod ./internal/lint -fix $$(go list ./... | grep -v '/acceptance/')
 	@go run -modfile tools/go.mod github.com/daixiang0/gci write -s standard -s default -s "prefix(github.com/hacbs-contract/ec-cli)" .
 
+.PHONY: tekton-lint
+tekton-lint: ## Run tekton-lint for 'tasks' subdirectory.
```

Comment:
Should have a list of dependencies, something like:
```suggest
tekton-lint: $(wildcard tasks/*/*/*/*.yaml)
```

The `tekton-lint` target should be included in the `lint` target.